# This method produce a traceback error, because the Python count starts at zero
# asking2much =["one","two","three"]
#  print(asking2much[3])

asking2much =["one","two","three"]
print(asking2much[2])

asking2much =["one","two","three"]
print(asking2much[-1])